### PR TITLE
CS - Czech language translate

### PR DESCRIPTION
**Pull request by No898**
Just added CS - Czech language ;)

### CODE REVIEW

File: loc/ui/messages.js [hunk 1]
```diff
@@ -70,6 +73,7 @@ export default {
     tr: '$1 klanına katılma isteği gönderdi.',
     ja: '$1 がクラン入隊の志願を送りました。',
     ko: '$1 이(가) 클랜에 참여하고 싶어합니다.'
```
Comment (timebomb0) [inline loc/ui/messages.js:75]: Missing trailing comma

File: loc/ui/messages.js [hunk 2]
```diff
@@ -105,7 +110,8 @@ export default {
     sr: 'Клан роле од $1 је демотиран.',
     tr: '$1 klan rütbesi düşürüldü.',
     ja: '$1 のクランロールが降格しました。',
-    ko: '$1 이(가) 강등되었습니다.'
+    ko: '$1 이(가) 강등되었습니다.',
```
Comment (timebomb0) [inline loc/ui/messages.js:113]: Missing trailing comma

File: loc/ui/messages.js [hunk 3]
```diff
@@ -196,6 +207,7 @@ export default {
     tr: '$1 partyden ayrıldı.',
     ja: '$1 がpartyを去りました。',
     ko: '$1 이 Party를 떠났습니다.'
```
Comment (timebomb0) [inline loc/ui/messages.js:209]: Missing trailing comma

File: loc/ui/messages.js [hunk 4]
```diff
@@ -52,6 +54,7 @@ export default {
     tr: '$1 klanından ayrıldı.',
     ja: '$1 がクランを去りました。',
     ko: '$1 이 클랜을 떠났습니다.'
```
Comment (timebomb0) [inline loc/ui/messages.js:56]: Missing trailing comma

File: loc/ui/messages.js [hunk 5]
```diff
@@ -298,10 +315,11 @@ export default {
     vi: 'Bạn đã bị đá ra khỏi Party $1.',
     zh: '你被 $1 從 Party 踢了。',
     sr: '$1 te je izbacio iz party-ja.',
-    it: 'Sei stato cacciato dal party da $2.',
+    it: 'Sei stato cacciato dal party da $1.',
```
Comment (timebomb0) [inline loc/ui/messages.js:318]: Calling this out for other reviewers to see. Looks like an unrelated-to-Czech Italian bugfix. `$2` to `$1` to align with the other strings. Nice.

Comment (No898) [reply loc/ui/messages.js:318]: Thank you.
